src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -170,6 +170,13 @@ export default class ComparisonStore {
     protected async saveAndGoToSession(newSession: Session) {
         throw new Error(`saveAndGoToSession must be implemented in subclass`);
     }
+    abstract _session: MobxPromise<Session>;
+    abstract _originalGroups: MobxPromise<ComparisonGroup[]>;
+    abstract overlapStrategy: OverlapStrategy;
+    abstract usePatientLevelEnrichments: boolean;
```

Comment:
the new typescript catches if the superclass declares an instance property and the subclass overrides it using a `get` accessor, so after some difficulty I realized that the solution is to declare these as abstract properties

Reply:
abstract forces the subclass to define it?

Reply:
I believe so, yes. and it allows the property to be implemented using an instance property or a getter/setter